src/app/public/modules/page-summary/page-summary-title.component.ts

```diff
@@ -1,5 +1,8 @@
 import { Component } from '@angular/core';
 
+/**
+ * Specifies a title in the summary to identify the page content.
+ */
```

Comment:
@Blackbaud-ToddRoberts, are we good with removing the qualifying info about title and subtitle: `You can pull data for the title from multiple sources, and you can combine multiple pieces of data in the title. The data to use depends on your users and the context in which they visit the page. You can display additional information in the subtitle. For example, you can display a record's natural language name in the title and its system-generated or coded identifier in the subtitle.`? Or should we port this content over to the Design tab?